packages/manager/modules/exchange/src/account/account.service.js

```diff
@@ -78,17 +81,12 @@ export default class ExchangeAccount {
    * @param {object} newAccount
    */
   sendingNewAccount(organizationName, serviceName, newAccount) {
-    return this.OvhHttp.post(
-      `/email/exchange/${organizationName}/service/${serviceName}/account`,
-      {
-        rootPath: 'apiv6',
-        data: newAccount,
-      },
-    ).then((data) => {
-      this.wucExchange.refreshViews('Accounts', 'Tasks');
-
-      return data;
-    });
+    return this.$http
+      .post(
+        `/email/exchange/${organizationName}/service/${serviceName}/account`,
+        newAccount,
+      )
+      .then(({ data }) => data);
```

Comment:
No need to call `this.wucExhange.refreshViews`?

Reply:
After creating an account the page is well refreshed and you arrive on the table with all the accounts.